### PR TITLE
Fix annoying visual bug with settings 

### PR DESCRIPTION
**Pull request by thibaultduponchelle**
I noticed the same bug than @choroba : https://github.com/kyzn/PRC/issues/31

I have no idea what I'm doing (what is detach() ?) but it seems to fix the bug :grinning: 

Edit : Tested locally

### CODE REVIEW

**Comment by kyzn** [comment]
Hi! Thanks a lot for your time 🙂 This looks good to me, but let me merge & get back to you next week.

**Comment by thibaultduponchelle** [comment]
It is just a small update :grinning: thank you for this website, it is very cool !

**Comment by kyzn** [comment]
I left a comment in #31 and tried to tag you there as well. I couldn't confirm the bug.. Can you possibly share your browser information, and step by step repro instructions?

File: lib/PRC/Controller/User.pm [hunk 1]
```diff
@@ -226,6 +229,9 @@ sub settings :Path('/settings') :Args(0) {
       my $selected_langs = $languages_form->values->{lang_select};
       $user->update_langs($selected_langs);
       $c->stash->{alert_success} = 'Your preferred languages are updated.';
```
Comment (kyzn) [inline lib/PRC/Controller/User.pm]: Can we convert these to session variables? Now that we are adding a reload, stash will be cleared by the time user lands on the page & the message will be missing. Session variable will be there for a little longer. Something like `$c->session->{alert_success}` in `sub skip_confirm`.

**Comment by kyzn** [comment]
So I've tried to use GitHub UI to leave some feedback, but I don't think it came out as I expected. In any case, this fix looks good to me! I just wanted to point to two things:
- There are 6 forms loaded in `/settings`. 2 of them already had reload, and your patch adds reload to other 2. I'd like all of them do have reload so this issue doesn't come back.
- In those 4 forms that don't reload, we were passing success message to stash. With the reload added, this will cause message to not show up. We should switch those to use session instead.

I truly appreciate you spending your time on fixing this issue! 🙂

**Comment by thibaultduponchelle** [comment]
I updated the PR with a new commit taking account of your feedback. 

I tested locally and checked that alerts are well printed.

Do I missed something ?

**Comment by kyzn** [comment]
> I tested locally and checked that alerts are well printed.

Huh, that's weird. When I was locally testing, alerts were not showing up. I wonder if it was a browser thing (I was on chrome).

With your latest commit it's working for me too. I think this is great now! 🙂 I'll be merging in a moment. Thanks so much for your time & help!

**Comment by thibaultduponchelle** [comment]
Great thank you 😃

**Comment by kyzn** [comment]
Thank _you_! I've also added your GitHub name to README.md/contributors. If you want me to list a different name, or take it off the list feel free to let me know :)

**Comment by thibaultduponchelle** [comment]
Thank you 😃 

It is not that important but I prefer Thibault Duponchelle instead :)

**Comment by kyzn** [comment]
Done! Thanks again.

**Pull request merged**